.rubocop_todo.yml

```diff
@@ -6,34 +6,6 @@
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 67
-# This cop supports safe autocorrection (--autocorrect).
-# Configuration parameters: Max, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, AllowedPatterns.
-# URISchemes: http, https
-Layout/LineLength:
-  Exclude:
-    - 'app/controllers/spree/users_controller.rb'
-    - 'app/models/customer.rb'
-    - 'app/models/spree/line_item.rb'
-    - 'spec/controllers/admin/order_cycles_controller_spec.rb'
-    - 'spec/controllers/spree/admin/orders_controller_spec.rb'
-    - 'spec/lib/reports/customers_report_spec.rb'
-    - 'spec/migrations/migrate_customer_names_spec.rb'
-    - 'spec/models/order_cycle_spec.rb'
-    - 'spec/models/product_importer_spec.rb'
-    - 'spec/models/spree/adjustment_spec.rb'
-    - 'spec/models/spree/line_item_spec.rb'
-    - 'spec/models/spree/variant_spec.rb'
-    - 'spec/services/products_renderer_spec.rb'
-    - 'spec/system/admin/adjustments_spec.rb'
-    - 'spec/system/admin/bulk_order_management_spec.rb'
-    - 'spec/system/admin/customers_spec.rb'
-    - 'spec/system/admin/order_cycles/simple_spec.rb'
-    - 'spec/system/admin/orders_spec.rb'
-    - 'spec/system/admin/product_import_spec.rb'
-    - 'spec/system/consumer/shopping/cart_spec.rb'
-    - 'spec/system/consumer/shopping/products_spec.rb'
-
```

Comment:
Something went wrong here. All these files are marked as fixed but this commit doesn't change them. In the next commit, this list is put back because they haven't been fixed. What happened?

Reply:
I was going to ask the same thing :p. These were generated by the script. It could be because I split the original PR #11261 since it had too many files. However, this was freshly generated on a new branch that incorporated the changes. I'll have to keep an eye out and squash these if I see them again.